### DISCUSSION TITLE
Working GSuite Domain as Alias Domain

### DISCUSSION THREAD

**Discussion by jdodz** [simple-login/app#1676] [Q&A]
Hello, Esteemed community!  I have read through other support requests trying to make sure I am not submitting a duplicate request.  I hope I am not here.  I did find this is a very active support community.  :) 

I am super excited about this concept.  I have 2 GSuite accounts, one I use in real estate, the other for other business.  I signed up for SimpleLogin with the "other business" email, and my plan was to create alias email addresses on the real estate email.  

My web host was able to add the txt record to the DNS (Cloudflare) then I verified ownership.  The MX records were next.  The web host added the MX records, this is the error I received:

> Your DNS is not correctly set. The MX record we obtain is:
> 
> 1 aspmx.l.google.com.
> 10 alt3.aspmx.l.google.com.
> 10 alt4.aspmx.l.google.com.
> 10 mx1.simplelogin.co.
> 20 mx2.simplelogin.co.
> 5 alt1.aspmx.l.google.com.
> 5 alt2.aspmx.l.google.com.

Obviously, the two MX records I was requested to add are there, but so are others, this is a working domain running on a GSuite Pro account.  Are the other MX records interfering with SimpleLogin?

Is there a workaround for this?

Thank you for your input.  :)

~ Jeff

**Comment by Practicalbutterfly5** [comment]
You can only have simplelogin or gsuite for mx record, not both. If you want you can create mx records for a subdomain instead.
